content/docs/integrations/kserve/KServe Ingress with kgateway.md

```diff
@@ -0,0 +1,138 @@
+
+---
+title: KServe Ingress with kgateway
+weight: 20
+---
+
+Use as the ingress gateway to manage traffic for your KServe machine learning inference services. This integration provides advanced routing, security, and observability capabilities for production ML workloads.
+
+### About kgateway with KServe
+kgateway provides a high-performance ingress solution for KServe that delivers comprehensive traffic management capabilities for machine learning workloads. As an ingress controller, it enables model-specific traffic routing through intelligent request distribution based on path patterns, headers, or model versions. The solution includes automatic TLS termination with certificate management for secure inference endpoints and configurable rate limiting to protect ML models from excessive traffic. For enterprise-grade security, kgateway supports advanced authentication methods including JWT validation, OAuth2 integration, and API key authentication, coupled with role-based authorization controls. Teams gain full observability through detailed request metrics, distributed tracing, and comprehensive access logging.
+
+### Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+
+### Additional requirements
+
+1. Verify KServe installation:
+   ```sh
+   kubectl get pods -n kserve-system
+   ```
+
+2. Install cert-manager for TLS certificates:
+   ```sh
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/latest/download/cert-manager.yaml
+   ```
+
+### Step 1: Configure basic ingress
+
+Create a VirtualService to route traffic to your KServe predictors:
+
+```yaml
+apiVersion: networking.kgateway.io/v1
+kind: VirtualService
```

Comment:
Why this guide using VirtualService and RouteTable instead of gateway API resources? (HTTPRoute, TrafficPolicy, etc.)